ui/raidboss/data/03-hw/dungeon/aetherochemical_research_facility.txt

```diff
@@ -101,11 +101,6 @@ hideall "--sync--"
 440.4 "Magitek Slug"
 444.0 "Magitek Slug"
 444.7 "Aetherochemical Grenado"
-449.9 "Aetherochemical Grenado"
```

Comment:
I'm not sure what you mean by "will never show up in the overlay".  The default is 30 seconds, and you can increase that arbitrarily.  So, even at the default, this would show 429 through 459?

Maybe I should file a feature request for this, but I think there'd be some advantage in having a new timeline entry that you could use instead of `jump` called ... `loop`? And if you did that, then the timeline code would automatically include this loop segment lookahead.

Reply:
That would normally be correct, yes, However, the max default is *also* six lines displayed at a time. My general impression has been that if either condition (30s of timeline, OR six lines) was reached, the rest could be removed. I'm fine not removing it, it was just a thing I came across in passing.

Reply:
You're correct as to when the rest are removed.  However, you can increase both of those limits, and so I've been doing minimum of 30 seconds (ignoring counts) for loops.

Reply:
Perfectly fair!